react/components/NumberOption.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from 'react'
+import { useAddressContext } from 'vtex.address-context/AddressContext'
+import { Input, Checkbox } from 'vtex.styleguide'
+import rules from '../countries/rules'
+import {
+  FormattedMessage,
+  injectIntl,
+  InjectedIntlProps,
+  defineMessages,
+} from 'react-intl'
+
+const messages = defineMessages({
+  wn: {
+    defaultMessage: '',
+    id: 'place-components.label.wn',
+  },
+})
+
+interface Props {
+  showCheckbox: boolean
+}
+
+const NumberOption: StorefrontFunctionComponent<Props & InjectedIntlProps> = ({
+  showCheckbox,
+  intl,
+}) => {
+  const { address, setAddress } = useAddressContext()
+  const [disabled, setDisabled] = useState(false)
+
+  const getInputProps = () => {
+    const field = rules[address.country].fields['number']
+    const maxLength = field && field.maxLength ? field.maxLength : null
+    const autoComplete = field && field.autoComplete ? field.autoComplete : null
+    const required = field && field.required ? field.required : null
+    const labelName = field && field.label ? field.label : null
```

Comment:
I think it is more idiomatic to write this and the next lines as
`const maxLength = field?.maxLength ?? null`

Reply:
Somehow my VSCode is not understanding that syntax of `?.` and it's breaking all other syntax checks, that's why I avoided it, I'll try again though.

Reply:
You might need to update the versions of the linting packages in your `package.json`. Try updating     `eslint`, `eslint-config-vtex-react` and `prettier` and see if VSCode still complaints.